core/src/main/java/org/jruby/ir/transformations/inlining/CFGInliner.java

```diff
@@ -15,12 +16,14 @@
 import java.util.List;
 
 public class CFGInliner {
-    private static final boolean debug = false;
+    private static final boolean debug = true;
```

Comment:
let's flip this back to `false` before :ship:

Reply:
Thanks for pointing this out.  I still am happy to see this output but I should use Log and hide behind the pref.  Just forgot to change this over. I will do that before merging.